Header header
uint16 remainTime;
uint8 gameStatus;
uint8 robotLevel;
uint16 remainHealth;
uint16 fullHealth;